src/dlgprefcontrols.cpp

```diff
@@ -89,25 +89,35 @@ DlgPrefControls::DlgPrefControls(QWidget * parent, MixxxMainWindow * mixxx,
     connect(ComboBoxRateDir, SIGNAL(activated(int)),
             this, SLOT(slotSetRateDir(int)));
 
-    // Set default range as stored in config file
-    if (m_pConfig->getValueString(ConfigKey("[Controls]", "RateRange")).length() == 0)
-        m_pConfig->set(ConfigKey("[Controls]", "RateRange"),ConfigValue(2));
-
     ComboBoxRateRange->clear();
-    ComboBoxRateRange->addItem(tr("6%"));
-    ComboBoxRateRange->addItem(tr("8% (Technics SL-1210)"));
-    ComboBoxRateRange->addItem(tr("10%"));
-    ComboBoxRateRange->addItem(tr("20%"));
-    ComboBoxRateRange->addItem(tr("30%"));
-    ComboBoxRateRange->addItem(tr("40%"));
-    ComboBoxRateRange->addItem(tr("50%"));
-    ComboBoxRateRange->addItem(tr("60%"));
-    ComboBoxRateRange->addItem(tr("70%"));
-    ComboBoxRateRange->addItem(tr("80%"));
-    ComboBoxRateRange->addItem(tr("90%"));
+    ComboBoxRateRange->addItem(tr("4% (Denon DN-D4500)"), 4);
+    ComboBoxRateRange->addItem(tr("6% (Pioneer CDJ/DDJ-S1)"), 6);
+    ComboBoxRateRange->addItem(tr("8% (Technics SL-1210, Pioneer DDJ-SX)"), 8);
+    ComboBoxRateRange->addItem(tr("10% (Pioneer CDJ/DDJ-S1, Denon DN-D4500)"), 10);
+    ComboBoxRateRange->addItem(tr("16% (Pioneer CDJ/DDJ-SX/DDJ-S1, Denon DN-D4500)"), 16);
+    ComboBoxRateRange->addItem(tr("24% (Denon DN-D4500)"), 24);
+    ComboBoxRateRange->addItem(tr("50% (Pioneer DDJ-SX, Denon DN-D4500)"), 50);
+    ComboBoxRateRange->addItem(tr("100% (Denon DN-D4500)"), 100);
```

Comment:
This allows stopping the deck with the rate slider. It is a nice feature. I am not sure if this works in all conditions.
We should stick with 90% for this pull request.